src/scss/_colors.scss

```diff
@@ -18,9 +18,9 @@
 
 // B2B theme
 @include oColorsSetUseCase('o-header-services-b2b-top-title', 'text', 'org-b2b-light');
-@include oColorsSetUseCase('o-header-services-b2b-primary-nav', 'background', 'org-b2b');
+@include oColorsSetUseCase('o-header-services-b2b-primary-nav', 'background', 'claret-1');
 @include oColorsSetUseCase('o-header-services-b2b-primary-nav', 'text', 'white');
-@include oColorsSetUseCase('o-header-services-b2b-primary-nav-item-hover', 'background', 'org-b2b-dark');
+@include oColorsSetUseCase('o-header-services-b2b-primary-nav-item-hover', 'background', 'claret-2');
```

Comment:
Would you mind opening an issue on o-colors to deprecate org-b2b-* colours? I think this was the only place they were used.